src/Dotnet.Script.Core/Commands/PublishCommand.cs

```diff
@@ -17,7 +17,7 @@ public PublishCommand(ScriptConsole scriptConsole, LogFactory logFactory)
             _logFactory = logFactory;
         }
 
-        public void Execute(PublishCommandOptions options)
+        public void Execute<TReturn>(PublishCommandOptions options)
```

Comment:
makes sense thanks. It's technically a breaking change - could you please keep the old one too and add the generic as a new overload?

It does not matter that much but I would like to have backwards compatibility in the public API unless we rev up major version

Reply:
Yes, you're right, should be an overload, backwards compatibility is very important. Sure, I'll add it